src/chrome/content/rules/FarManager.xml

```diff
@@ -0,0 +1,10 @@
+<ruleset name="FarManager">
+    <target host="farmanager.com" />
+    <target host="www.farmanager.com" />
+    <target host="forum.farmanager.com" />
+    <target host="plugring.farmanager.com" />
+    <target host="api.farmanager.com" />
+    <target host="bugs.farmanager.com" />
```

Comment:
Please also add `enforum.farmanager.com` and sort the targets alphabetically (you can keep `^farmanager.com` and `www.farmanager.com` on top).